Plugins/Json/include/Acts/Plugins/Json/JsonGeometryConverter.hpp

```diff
@@ -123,9 +123,9 @@ class JsonGeometryConverter {
     /// The data key
     std::string datakey = "data";
     /// The geoid key
-    std::string geoidkey = "Geoid";
+    std::string geometryidkey = "Geoid";
     /// The surface geoid key
-    std::string surfacegeoidkey = "SGeoid";
+    std::string surfacegeometryidkey = "SGeoid";
```

Comment:
I know we hadn't really CamelCased this one ... should be anyway part of a different PR if we decide to do so.

Reply:
Yeah right, I only touched the spelling, but this should eventually also be changed to camelCase I think (/cc @Corentin-Allaire)

Reply:
Yeah, the key used for the json map don't use CamelCased. Not sure why to be honest but I could definitely open a PR to change that. Not really sure we need to replace surfacegeoidkey by surfacegeometryidkey but since you have already done it I guess that is fine.